README.md

```diff
@@ -2,6 +2,8 @@
 
 [![codecov](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses/branch/main/graph/badge.svg?token=LvEIQ6tYti)](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses)
 
+* Team 1 Heroku QA Link: <https://s22-5pm-1-courses-qa.herokuapp.com/>
+
```

Comment:
Missing a link to the heroku dashboard for the qa deployment. Make sure it is shared with all of the staff as well. Also please add a description to the pull request. Thanks!

Reply:
Changes fixed

Reply:
Still missing a link to the dashboard and pull request description

Reply:
Ok i think i fixed it now